### PR TITLE
Package details

### PR DESCRIPTION
**Pull request by jeroen**
Adds a collapsible details pane to the package table, including citations. Also some tweaks and cleanups.

### CODE REVIEW

**Comment by maelle** [comment]
Niiice!!

**Comment by maelle** [comment]
@jeroen would it make sense to make citations and description available to search?

**Comment by jeroen** [comment]
Hmm not sure, maybe that leads to a lot of search overhead and false positives?

**Comment by maelle** [comment]
It is a trade-off for sure. 🤔

**Comment by jeroen** [comment]
Actually the description is already searchable I think. It always has been. We have a hidden field in the table just for that.

**Comment by maelle** [comment]
Ok but citations aren't... probably fine, since keywords are, and hopefully package authors use them (codemetar uses e.g. repo topics).

Tl;dr no further comment.
